12_cookies.html

```diff
@@ -104,22 +104,22 @@
               <td></td>
             </tr>
           </table>
-          <p>A command named "addCookie" MUST follow:
+          <p>The <a>remote end steps</a> for the <dfn>Add Cookie</dfn> command are:</p>
           <ol>
-            <li>If the parameters object does not contain an element that can be coverted to
-              <code><a>Cookie</a></code> return a <code><a href='#status-unable-to-set-cookie'>unable to set cookie</a></code> error.</li>
-            <li>If the <code>Document</code> is a <a href='http://www.w3.org/html/wg/drafts/html/master/single-page.html#cookie-averse-document-object'>cookie-averse <code>Document</code> object</a> then return an
-              <code><a href="#status-invalid-cookie-domain">invalid cookie domain</a></code> error.</li>
-            <li>Set the <a>Cookie</a> in the user agent following the steps described in <a href='http://tools.ietf.org/html/rfc6265#section-5.3'>Storage Model</a> in [[!RFC6265]]. If there is an error during this step
-              return a <code><a href='#status-unable-to-set-cookie'>unable to set cookie</a></code> error.</li>
+            <li><p>If the <a>current browsing context</a> is <a>no longer open</a>,
+                return an error with code <a>no such window</a>.</p></li>
+            <li><p>Let <var>cookie</var> be the result of <a>getting a property</a>
+              named "cookie" from the <var>parameters</var> argument. If <var>cookie</var>
+              is not a map then return an error <a>unable to set cookie</a>.</p></li>
+            <li><p>If the <code>Document</code> is a <a href='http://www.w3.org/html/wg/drafts/html/master/single-page.html#cookie-averse-document-object'>
+              cookie-averse <code>Document</code> object</a> then return an error with code
+              <a>invalid cookie domain</a>.</p></li>
+            <li><p>Store the contents <var>cookie</var> in the user agent cookie manager
```

Comment:
Not sure if this section needs to have a mapping description to move things from the `map` to a format that can be stored in [rfc6265](http://tools.ietf.org/html/rfc6265#section-5.3)